Products/ZenUI3/browser/resources/js/zenoss/zenoss.js

```diff
@@ -68,6 +68,80 @@ Ext.namespace('Zenoss.env');
 
 Ext.QuickTips.init();
 
+Ext.enableGarbageCollector = false;
+Zenoss.env.enableGarbageCollector = true;
+
```

Comment:
This workaround is the full copy of current ExtJs GC function in Zenoss except lines with deprecated function. The reason I did it is because in new versions of ExtJs I didn't find fix of deprecated calls in GC function